### ISSUE TITLE
Add Missing Import Quick Fix Should Be able to use File/Module Name for Default Exports

### ISSUE THREAD

**Issue by mjbvz** [Microsoft/TypeScript#14663]
From https://github.com/Microsoft/vscode/issues/21764#issuecomment-286404917

**TypeScript Version:**  2.2.1 

Example one in a project with the `absolute ` module installed, write:

```ts
absolute('/dir/file');
```

There is no quick fix for adding a import like: `import absolute from 'absolute'`

---

Example Two:

In a project that uses default exports:

```ts
// abc.ts
export default 'abc'
```

```ts
// other.ts
console.log(abc);
```

No  quick fix is generate for importing `abc` from `abc.ts`. If the default exported symbol has a name, such as when using a class or function, then we can import it.

**Comment by mhegazy** [comment]
We really need to put the symbol in the completion list first.

**Comment by mjbvz** [comment]
Was this fixed by #19875?

**Comment by andy-ms** [comment]
Yes, included a test in `importNameCodeFixDefaultExport.ts` and just verified myself using `typescript@next`.
